scipy/optimize/tests/test_optimize.py

```diff
@@ -290,6 +290,19 @@ def test_powell_bounded(self):
             assert self.funccalls <= 131 + 20
             assert self.gradcalls == 0
 
+    def test_warn_neldermead(self):
+        with pytest.warns(UserWarning, match=r'Maximum number of function'):
+            func = lambda x: np.sum(x ** 2)
+            p0 = [0.15746215, 0.48087031, 0.44519198, 0.4223638, 0.61505159,
```

Comment:
Why these specific values?

Reply:
I agree - we could use much simpler initial values with a really low function evaluation limit. Do you agree?